docs/components/board/customlinux.md

```diff
@@ -9,11 +9,27 @@ tags: ["board", "components"]
 # SMEs: Susmita
 ---
 
-Configure a `customlinux` board to integrate linux boards like the [Mediatek Genio 500 Pumpkin](https://ologicinc.com/portfolio/mediateki500/) into your robot.
+The `customlinux` board model supports boards like the [Mediatek Genio 500 Pumpkin](https://ologicinc.com/portfolio/mediateki500/) that run Linux operating systems and are not supported by other built-in Viam models.
+
+To integrate a custom Linux board into your robot:
+
+1. First [create a pin mappings file](#create-a-pin-mappings-file).
+2. Then [configure a `customlinux` board](#configure-your-board) on your robot, specifying the path to the mappings file in the board config.
 
 ## Create a pin mappings file
 
-On your `customlinux` board, create a file in your <file>/home/root</file> directory called <file>board.json</file> with your board's pin mappings:
+{{% alert title="Caution" color="caution" %}}
+
+While some lines on a board are attached to GPIO pins, some lines are attached to other board hardware.
+It is important to carefully determine your `line_number` values.
+Randomly entering numbers may result in hardware damage.
+
+{{% /alert %}}
+
+The pin mappings file describes the location of each GPIO pin on the board so that `viam-server` can access the pins correctly.
+
+On your `customlinux` board, create a file in your <file>/home/root</file> directory called <file>board.json</file> with your board's pin mappings.
```

Comment:
This is fine as an example, but the file can go anywhere else, too. You could put it in `/tmp` or `/opt` or `/etc` or `/home/nonroot_user` or anything. You can also name it anything you want, such as `mediatek_pumpkin.json` or `backyard_weeder_v2.json`.

Reply:
```suggestion
On your `customlinux` board, create a JSON file in the directory of your choice with your board's pin mappings.
```